t/app/controller/waste_kands_bulky.t

```diff
@@ -401,9 +401,13 @@ FixMyStreet::override_config {
         };
 
         subtest 'Confirmation page' => sub {
-            $mech->content_contains('Payment successful');
-
             $report = FixMyStreet::DB->resultset("Problem")->search(undef, { order_by => { -desc => 'id' } })->first;
+            $mech->content_contains('Bulky collection booking confirmed');
+            $mech->content_contains('Our contractor will collect the items you have requested on 08 July.');
+            $mech->content_contains('Item collection starts from 6:30am. Please have your items ready and dismantled if required.');
+            $mech->content_contains('We have emailed confirmation of your booking to pkg-tappcontrollerwaste_kands_bulkyt-bob@example.org.');
+            $mech->content_contains('If you need to contact us about your application please use the application reference:
+          RBK-' . $report->id);
```

Comment:
This feels a bit brittle - could we use content_like and a regex with `\s` instead, perhaps? Or adjust the template brackets so that it's only one space :)